portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,91 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>_LUJIE_</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="script.js"></script>
   </head>
+
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <h1>LUJIE ZHAO</h1>
+
+      <section class="details">
+        <div id="left-details">
+            <p>"Keep your head in the cloud."</p>
+            <img src="/home/lujiez/headshot.JPG" alt="headshot">
+        </div>
+        <div id="right-details">
+            <i style="font-size:24px" class="fa">&#xf08c;</i><a href="https://www.linkedin.com/in/lujie-sue-zhao/" style="color: white">linkedin.com/in/lujie-sue-zhao</a>
+            <br>
+            <i style="font-size:24px" class="fa">&#xf092;</i><a href="https://github.com/LujieZ" style="color: white">github.com/LujieZ </a>
+        </div>
+        <div class="clear"></div>
+      </section>
+
+      <section class="area">
+        <div id="display-container">
+            <h2>About</h2>
+            <p> A sophomore BS student studying Computer Science
+                at NYU Tandon, minors in Business Studies and Finance. 
+                Currently interested in pursuing software/finance internship
+                    opportunities. </p>
```

Comment:
A nicer way to format paragraphs elements:

```
<p>
  A sophomore BS student studying Computer Science at NYU Tandon,
  minors in Business Studies and Finance. Currently interested in
  pursuing software/finance internship opportunities.
</p>
```

Reply:
Format for the paragraph is adjusted.

Reply:
If `</p>` is in a separate line then it will look nicer if `<p>` does the same.